packages/muffincss-core/src/processors/rules.ts

```
@@ -2,27 +2,17 @@ import { type AtRule, type Declaration, type Root } from "postcss";
     options: Options["options"],
   ) {
     super(resultCollector, options);
-    this.parsedRules = new Map<string, AtomicRule>();
-  }
-  private addToParsedRules(className: string, declaration: Declaration) {
-    if (!this.parsedRules.has(className))
-      this.parsedRules.set(className, {
-        prop: declaration.prop,
-        value: declaration.value,
-      });
   }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)